src/main/kotlin/lotto/domain/LottoNumbers.kt

```diff
@@ -17,6 +17,11 @@ class LottoNumbers(
         }
     }
 
+    fun matches(inputNumber: LottoNumber): Boolean {
+        val matchingNumber = numbers.firstOrNull { it.matches(inputNumber) }
+        return matchingNumber != null
+    }
```

Comment:
```suggestion
    fun matches(inputNumber: LottoNumber): Boolean {
        return numbers.contains(inputNumber)
    }
```
간단하게 구현할 수 있을 것 같습니다.